project1/styles.css

```diff
@@ -1,5 +1,67 @@
+body {
+    background-color: hsl(254, 0%, 100%);
+    border: 3px;
+}
+
+figure {
+    float: right;
+    margin-right: 40px;
+    padding: 30px;
+
+
+}
+
+iframe {
+    margin: 60px;
+}
+
 img {
-    max-width: 100%
+    max-width: 100%;
 }
 
-/* comments look like this */
+figcaption {
+    color: hsl(0, 0%, 40%);
+    font-family: 'amiri', serif;
+}
+
+h1 {
+    color: hsl(299, 32%, 60%);
+    text-align: center;
+    font-family: 'amiri', serif;
+    font-size: 48px;
+    font-weight: normal;
+    margin: 30px;
+}
+
+h2 {
+    color: hsl(0, 0%, 40%);
+    text-align: center;
+    font-family: 'amiri', serif;
+    font-size: 21px;
+    font-weight: normal;
+    margin: 30px;
+}
+
+p {
+    color: hsl(0, 0%, 40%);
+    font-family: 'quicksand', serif;
+    font-size: 18px;
+    margin: 60px;
```

Comment:
I think these margins are too wide for body text that relates to each other. You want paragraphs to be distinguishable from each other, but not to have wide gaps between them (for the most part).